tests/pass-dep/libc/libc-epoll.rs

```diff
@@ -165,7 +166,9 @@ fn test_epoll_ctl_del() {
     assert_ne!(res, -1);
 
     // Test EPOLL_CTL_DEL.
-    check_epoll_wait::<0>(epfd, &[]);
+    let res = unsafe { libc::epoll_ctl(epfd, libc::EPOLL_CTL_DEL, fds[1], &mut ev) };
+    assert_eq!(res, 0);
+    check_epoll_wait::<8>(epfd, &[]);
```

Comment:
This is a duplicate of #3813 to let all the testcases pass.

Reply:
Okay, please rebase once https://github.com/rust-lang/miri/pull/3813 landed then.